server/service/src/sync/translations/invoice.rs

```diff
@@ -650,8 +650,7 @@ fn invoice_status(invoice_type: &InvoiceType, data: &LegacyTransactRow) -> Optio
             LegacyTransactStatus::Fn => InvoiceStatus::Verified,
             _ => return None,
         },
-        // mSupply will alert users to finalise any repacks if they have un-finalised repacks
-        // before they migrate to Open mSupply.
+        // mSupply will delete any unfinalised repacks before migration
```

Comment:
Change comment since OG is now deleting unfinalised repacks instead of alerting user